sdks/jreleaser-git-java-sdk/src/main/java/org/jreleaser/sdk/git/ChangelogGenerator.java

```diff
@@ -597,30 +617,135 @@ private void addContributor(String name, String email) {
+                scope = matcherFirstLine.group("scope") == null ? "" : matcherFirstLine.group("scope");
+                description = matcherFirstLine.group("description");
+            } else {
+                isConventional = false;
+                return;
+            }
+
+            // drop any empty lines at the beginning
+            while (!lines.isEmpty() && lines.get(0).equals("")) {
+                lines.remove(0);
+            }
+
+            // try to match trailers from the end
+            while (!lines.isEmpty()) {
+                Matcher matcherTrailer = TRAILER_PATTERN.matcher(lines.get(lines.size() - 1));
+                if (matcherTrailer.matches()) {
+                    String token = matcherTrailer.group("token");
+                    if(token.equals("BREAKING-CHANGE")) break;
+                    trailers.add(new Trailer(token, matcherTrailer.group("value")));
+                    lines.remove(lines.size() - 1); // consume last line
+                } else {
+                    break;
+                }
+            }
+
+            // drop any empty lines at the end
+            while (!lines.isEmpty() && lines.get(lines.size() - 1).equals("")) {
+                lines.remove(lines.size() - 1);
+            }
+
+            Matcher matcherBC = BREAKING_CHANGE_PATTERN.matcher(String.join("\n", lines));
+            if (matcherBC.find()) {
+                isBreakingChange = true;
+                breakingChangeContent = matcherBC.group("content");
+                // consume the breaking change
+                OptionalInt match = IntStream.range(0, lines.size())
+                    .filter(i -> BREAKING_CHANGE_PATTERN.matcher(lines.get(i)).find())
+                    .findFirst();
+                if (match.isPresent()) {
+                    if (lines.size() > match.getAsInt()) {
+                        lines.subList(match.getAsInt(), lines.size()).clear();
+                    }
+                }
+            }
+
+            // the rest is the body
+            body = String.join("\n", lines);
+        }
+
+        public static Commit of(RevCommit rc) {
+            ConventionalCommit c = new ConventionalCommit(rc);
+            if(c.isConventional) return c;
+                // not ideal to reparse the commit, but that way we return a Commit instead of a ConventionalCommit
+            else return Commit.of(rc);
+        }
+
+        @Override
+        Map<String, Object> asContext(boolean links, String commitsUrl) {
+            Map<String, Object> context = super.asContext(links, commitsUrl);
+            context.put("commitIsConventional", true);
+            context.put("commitBreakingChangeContent", passThrough(breakingChangeContent));
+            context.put("commitIsBreakingChange", isBreakingChange);
+            context.put("commitType", type);
+            context.put("commitScope", scope);
+            context.put("commitDescription", description);
```

Comment:
may need `passThrough()` as well otherwise Mustache might escape content

Reply:
`isBreakingChange` is a boolean, but i will change the other 3